docs/model-dev-guide/dtrain/dtrain-implement.rst

```diff
@@ -24,110 +25,117 @@ training, as described in :ref:`cluster-configuration`.
-larger value enables multi-GPU training automatically. Note that these GPUs might be on a single
-machine or across multiple machines; the experiment configuration simply defines how many GPUs
-should be used for training, and the Determined job scheduler decides whether to schedule the task
-on a single agent or multiple agents, depending on the machines in the cluster and the other active
-workloads.
+By default, this field is set to a value of ``1``, which essentially disables distributed training.
+If you increase the ``slots_per_trial`` value, this will automatically enable multi-GPU training.
+Bear in mind that these GPUs can either be located on a single machine or distributed across
+multiple machines. The experiment configuration merely dictates the number of GPUs to be used in the
+training process, while the Determined job scheduler decides whether to schedule the task on a
+single agent or multiple agents. Whether to schedule the task on a single agent or multiple agents
+depends on the machines in the cluster and other active workloads.
 
-Multi-machine parallelism offers the ability to further parallelize training across more GPUs. To
-use multi-machine parallelism, set ``slots_per_trial`` to be a multiple of the total number of GPUs
-on an agent machine. For example, if your resource pool consists of multiple 8-GPU agent machines,
-valid values for ``slots_per_trial`` would be 16, 24, 32, etc. In this configuration, trials use all
-the resources of multiple machines to train a model:
+Multi-machine parallelism allows you to further parallelize training across more GPUs. To use this
+feature, ``slots_per_trial`` should be set as a multiple of the total number of GPUs on an agent
+machine. For example, if your resource pool consists of multiple 8-GPU agent machines, valid
+``slots_per_trial`` values would be 16, 24, 32, and so on.
+
+In the following configuration, trials will use the combined resources of multiple machines to train
+a model:
 
 .. code:: yaml
 
    resources:
      slots_per_trial: 16  # Two 8-GPU agent machines will be used in a trial
 
-For distributed multi-machine training, Determined automatically detects a common network interface
-shared by the agent machines. If your cluster has multiple common network interfaces, please specify
-the fastest one in :ref:`cluster-configuration` under
+For distributed multi-machine training, Determined will automatically detect a common network
+interface that is shared by the agent machines. If your cluster has multiple common network
+interfaces, we advise specifying the fastest one in :ref:`cluster-configuration` under
 ``task_container_defaults.dtrain_network_interface``.
 
-Reduce computational overhead by setting the ``global_batch_size`` to the largest batch size that
-fits into a single GPU multiplied times the number of slots.
+You can reduce computational overhead by setting the ``global_batch_size`` to the largest batch size
```

Comment:
do we need this sentence here since we already have a section right below this for global batch size?

Reply:
for discussion: where does the user start thinking about or need to know about reducing computational overhead?

Reply:
resolved, moved the sentence into the section about global batch size